src/client.js

```diff
@@ -75,7 +75,7 @@ let getRegistration = async function() {
 
 class PushClient {
   constructor({endpointUrl=null, userId=null, workerUrl=WORKER_URL,
-      scope=SCOPE}) {
+      scope=SCOPE} = {}) {
```

Comment:
What does this change do? I'm new to the syntax and this looks like it's setting a default for no object passed in to be an empty object.